java/DynamoDBStreamSource/README.md

```diff
@@ -0,0 +1,37 @@
+# DynamoDB Streams Source example
+
+* Flink version: 1.20
+* Flink API: DataStream API
+* Language: Java (11)
+* Flink connectors: DynamoDb Streams Source
+
+
+This example demonstrate how to use Flink DynamoDB Streams source.
+
+This example uses `DynamoDbStreamsSource`.
+
```

Comment:
Explain the basic requirements and how the user can make this work:
1. A DDB table - document the schema
2. Set up DDBStreams - link do docs
3. Make changes to the DDB table - e.g. via console

Explain what permissions the app must have to read from DDBstream (and to write to the Kinesis sink)

Reply:
Done!